apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationCard.tsx

```diff
@@ -56,6 +56,7 @@ const ApplicationCard = ({
     application.poolAdvertisement?.closingDate,
     intl,
   );
+  const status = getPoolCandidateStatusLabel(application.status);
```

Comment:
A little out of scope, but I'm worried about the `isExpired` function above, online 51.  It looks like it's using `parseISO` and I'm concerned that it's parsing a UTC date as a local one.  Could you double-check if it should be using `parseDateTimeUtc` instead?

Reply:
Good catch! You are right about that, fixed in [update application expiry to parse to UTC](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5798/commits/6c4deb242b131caa895428f2e815d82644b0991f)